### ISSUE TITLE
Editor

### ISSUE THREAD

**Issue by rzr** [rzr/pinball#11]
Should we resurect pinedit ?
or think about exporting from 3D tools ?
I am considering about using GLTf format

More motes at:
https://gist.github.com/freezy/37ea49bc2121073b57c65f1b17f3fc40

@sergiomb2

**Comment by jgengelhardt** [comment]
Resurrecting pinedit seems to be the simplest first step for enabling designers to contribute more content to the project. I don't think a new editor would see completion.

**Comment by rzr** [comment]
Yes I think using containers it's something that would not require porting effort

https://snapcraft.io/pinedit
